apisix/utils/redis-schema.lua

```diff
@@ -42,6 +42,9 @@ local policy_to_additional_properties = {
             redis_ssl_verify = {
                 type = "boolean", default = false,
             },
+            sync_interval = {
+                type = "integer", default = -1
```

Comment:
default to 0 and with restriction >= 0?

Reply:
I don't get you. But the default value denotes normal mode of operation. i.e sync with redis immediately.

Reply:
Because `delay(<=0)` arg. to the `ngx.timer.at` function means executing the timeout callback as soon as the current handler yield.

But negative number for `interval` have no meaning, so restrict `sync_interval >= 0`. 
And sync_interval = 0(default) means no delay(sync with redis immediately).

Reply:
sync_interval is in seconds, 0 second would gives misleading idea of "sync immediately"